src/components/ProfilePicker/ProfilePickerReferenceWidget.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+	<div class="profile-reference">
+		<div class="profile-reference__wrapper">
+			<div class="profile-card__header">
+				<NcAvatar :user="richObject.user_id" :size="48" class="profile-card__avatar" />
+				<div class="profile-card__title">
+					<a :href="richObject.url" target="_blank">
+						<Account :size="20" />
+						<strong>
+							{{ richObject.email !== null ? richObject.title + ' - ' + richObject.email : richObject.title }}
+						</strong>
+					</a>
+				</div>
+			</div>
+			<div class="profile-content">
+				<p class="profile-content__subline">
+					<span v-if="richObject.headline" class="headline">
+						{{ richObject.headline }}
+					</span>
+					<span v-if="richObject.location" class="location">
+						<MapMarker :size="20" />
+						<template v-if="richObject.location_url">
+							<a :href="richObject.location_url" class="link" target="_blank">{{ richObject.location }}</a>
+						</template>
+						<template v-else>
+							{{ richObject.location }}
+						</template>
+					</span>
+					<span v-if="richObject.website" class="website">
+						<Web :size="20" />
+						<a :href="richObject.website" class="link" target="_blank">{{ richObject.website }}</a>
+					</span>
+					<span v-if="richObject.organisation" class="organisation">
+						<Domain :size="20" />
+						{{ richObject.organisation }}
+					</span>
+					<span v-if="richObject.role" class="role">
+						<Handshake :size="20" />
+						{{ richObject.role }}
+					</span>
+					<span v-if="richObject.bio" class="bio">
+						<TextAccount :size="20" />
+						{{ richObject.bio }}
+					</span>
+				</p>
+			</div>
+		</div>
+	</div>
+</template>
+
+<script>
+import NcAvatar from '@nextcloud/vue/dist/Components/NcAvatar.js'
+
+import Account from 'vue-material-design-icons/Account.vue'
+import MapMarker from 'vue-material-design-icons/MapMarker.vue'
+import Web from 'vue-material-design-icons/Web.vue'
+import Domain from 'vue-material-design-icons/Domain.vue'
+import Handshake from 'vue-material-design-icons/Handshake.vue'
+import TextAccount from 'vue-material-design-icons/TextAccount.vue'
+
+export default {
+	name: 'ProfilePickerReferenceWidget',
+	components: {
+		NcAvatar,
+		Account,
+		MapMarker,
+		Web,
+		Domain,
+		Handshake,
+		TextAccount,
+	},
+	props: {
+		richObjectType: {
+			type: String,
+			default: '',
+		},
+		richObject: {
+			type: Object,
+			default: null,
+		},
+		accessible: {
+			type: Boolean,
+			default: true,
+		},
+	},
+}
+</script>
+
+<style scoped lang="scss">
+.profile-reference {
+	width: 100%;
+	white-space: normal;
+	display: flex;
+
+	.profile-reference__wrapper {
```

Comment:
```suggestion
	&__wrapper {
```

tip: makes it easier to work with BEM in scss